File: Dinner/Java/CheckPerfectNumber.java
import java.util.Scanner;

public class CheckPerfectNumber{
	public static void main(String[] args){

		Scanner scanner = new Scanner(System.in);
	
		System.out.print("Enter a number: ");
		int number = scanner.nextInt();
		int sum = 0;
		for(int count = 1; count <= number; count++){
			if(number % count == 0) {
			sum = sum + count;
				
			}
 		}

		int perfect = sum - number ;		
		if(perfect == number){
			System.out.println(number + " is a perfect number");
		}
		else{
			System.out.println(number + " is not a perfect number");
		}


	}


}

File: Dinner/Python/JavaScript/CheckPerfectNumber.js
let number = 6;
let sum = 0;
for(let count = 1; count <= number; count++){
	if(number % count == 0) 
		sum = sum + count;
				
 }
let perfect = sum - number ;		
if(perfect == number)
	console.log(number + " is a perfect number");
else
	console.log(number + " is not a perfect number");
		



